### PR TITLE
Update Classification notebook with new API

### PR DESCRIPTION
**Pull request by zwasd**
#4463

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -185,6 +185,9 @@ Session.vim
 *~
 tags
 
+# VS Code
+.vscode
+
```
Comment (gf712) [inline .gitignore]: Could you remove this please?

**Comment by gf712** [comment]
Looks good, did you test it locally?
Do you know how many aren't working?

File: .gitignore (hunk 1 above)
Comment (zwasd) [reply .gitignore]: I'm using vscode mainly, and don't want to accidentally commit the settings folder

**Comment by zwasd** [comment]
> Looks good, did you test it locally?

I've run it in jupyter notebook, all the code seems to work

File: .gitignore (hunk 1 above)
Comment (gf712) [reply .gitignore]: yes, that's fine, but you shouldn't push your local .gitignore as not everyone uses .vscode

Comment (zwasd) [reply .gitignore]: ok, i'll remove it

**Comment by vigsterkr** [comment]
@zwasd could you plz just push the commit that doesn't add the .gitignore changes, i'd really like to merge this into develop!

**Comment by zwasd** [comment]
@vigsterkr I've pushed them

**Pull request merged**